pkg/client/kubernetes/cache/single_object.go

```diff
@@ -148,6 +148,10 @@ func (s *singleObject) GetInformer(ctx context.Context, obj client.Object, opts
 	return cache.GetInformer(ctx, obj, opts...)
 }
 
+func (s *singleObject) RemoveInformer(_ context.Context, _ client.Object) error {
+	return nil
```

Comment:
Why is this not implemented? Can't we implement it in the same way like `GetInformer`?